examples/smartContracts/contracts/sc_code.ts

```diff
@@ -7,17 +7,72 @@ import {
   callerHasWriteAccess,
   Storage,
 } from '@massalabs/massa-as-sdk';
-import { Args, bytesToString, stringToBytes } from '@massalabs/as-types';
+import { Args, Result, Serializable, stringToBytes } from '@massalabs/as-types';
 
-const DATA_KEY = 'key';
-const DATA_VALUE = 'value';
+const MUSIC_ALBUM_KEY = 'MUSIC_ALBUM_KEY';
+
+class MusicAlbum implements Serializable {
+  constructor(
+    public id: string = '',
+    public title: string = '',
+    public artist: string = '',
+    public album: string = '',
+    public year: u64 = 0,
+  ) {}
+
+  public serialize(): StaticArray<u8> {
+    const args = new Args();
+    args.add(this.id);
+    args.add(this.title);
+    args.add(this.artist);
+    args.add(this.album);
+    args.add(this.year);
+    return args.serialize();
+  }
+
+  public deserialize(data: StaticArray<u8>, offset: i32 = 0): Result<i32> {
+    const args = new Args(data, offset);
+
+    const id = args.nextString();
+    if (id.isErr()) {
+      return new Result(0, "Can't deserialize the id");
+    }
+    this.id = id.unwrap();
+
+    const title = args.nextString();
+    if (title.isErr()) {
+      return new Result(0, "Can't deserialize the title");
+    }
+    this.title = title.unwrap();
+
+    const artist = args.nextString();
+    if (artist.isErr()) {
+      return new Result(0, "Can't deserialize the artist");
+    }
+    this.artist = artist.unwrap();
+
+    const album = args.nextString();
+    if (album.isErr()) {
+      return new Result(0, "Can't deserialize the album");
+    }
+    this.album = album.unwrap();
+
+    const year = args.nextU64();
+    if (year.isErr()) {
+      return new Result(0, "Can't deserialize the year");
+    }
+    this.year = year.unwrap();
+
+    return new Result(args.offset);
+  }
+}
 
 /**
  * This function is meant to be called only one time: when the contract is deployed.
  *
- * @param args - Arguments serialized with Args
+ * @param _args - Arguments serialized with Args
  */
-export function constructor(args: StaticArray<u8>): StaticArray<u8> {
+export function constructor(_args: StaticArray<u8>): StaticArray<u8> {
```

Comment:
Why use `_` which usually means `unused` on that variable since it is passed to another function at line 84?